spec/features/early_career_payments_eligible_now_reminder_set_spec.rb

```diff
@@ -99,7 +99,7 @@
             click_on "Continue"
             fill_in "reminder_one_time_password", with: get_otp_from_email
             click_on "Confirm"
-            expect(page).to have_text("We will send you a reminder in September #{claim.eligibility.eligible_later_year.start_year}")
+            expect(page).to have_text("We will send you a reminder in September #{(PolicyConfiguration.for(claim.policy).current_academic_year + 1).start_year}")
```

Comment:
You assigned the interpolated value to `reminder_year` in the spec above. Maybe you want to do the same here.

Reply:
Yeah it's nicer I'll change, I just changed what was there, that's a minor improvement :)